CaptureSnippets/IncludeExtracotrs/CSharpUsingExtractor.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Text.RegularExpressions;
+
+namespace CaptureSnippets.IncludeExtracotrs
+{
+    public class CSharpUsingExtractor : IIncludeExtractor
+    {
+        const string UsingPattern = @"(?:using\s)(?<ns>.*)(?:;)";
```

Comment:
i would much prefer you use normal string manipulation instead of a regex. it will be faster, use less mem and less likely to have bugs